src/test/java/com/puppycrawl/tools/checkstyle/TreeWalkerTest.java

```diff
@@ -195,7 +206,12 @@ public void testWithCacheWithNoViolation() throws Exception {
         final File file = temporaryFolder.newFile("file.java");
         final List<String> lines = new ArrayList<>();
         lines.add(" class a {} ");
-        treeWalker.processFiltered(file, lines);
+        try {
+            treeWalker.processFiltered(file, lines);
+        }
+        catch (CheckstyleException ex) {
+            fail("Exception is not expected");
```

Comment:
@romani, ok, the best choice is to rewrite them. In any case it will help pitest mutation coverage to increase.